frontend/src/components/common/AddAdminModal.tsx

```diff
@@ -154,38 +153,34 @@ const AddAdminModal = (): React.ReactElement => {
               <ModalCloseButton />
               <ModalBody>
                 {showRequestError && (
-                  <Alert
-                    status="error"
-                    mb={10}
-                    variant="top-accent"
-                    borderColor="red.200"
-                    backgroundColor="red.50"
-                    color="red.200"
-                    textStyle="mobileSubtitle1"
-                  >
-                    <AlertIcon />
+                  <Alert status="error" mb={10} borderColor="red.200">
```

Comment:
`borderColor` is specified here because of a known Chakra UI issue where border stying is ignored if specified in the theming: https://github.com/chakra-ui/chakra-ui/issues/2579